src/api/mocks.ts

```diff
@@ -10,12 +10,13 @@ async function loadMockFile(importFn: () => Promise<any>): Promise<any> {
 
 function getBaseInfo(): Info {
   return {
-    hostname: 'Tomaszs-MacBook-Pro-16-inch-2019',
+    hostname: 'control-plane-5d94cb99c6-rzr96',
     tagline: import.meta.env.VITE_NAMESPACE,
     version: '1.7.1',
```

Comment:
nitpick: we could just update to 2.0.0 😉

Reply:
Sure, we could, but it’s not really relevant. It only affects using mock data and gives us a fuzzy way to see if URL links to older docs sources still work.